cmd/manager/main.go

```diff
@@ -64,11 +60,8 @@ import (
 )
 
 const (
-	defaultLogLevel             = "info"
-	leaderElectionConfigMap     = "hive-controllers-leader"
-	leaderElectionLeaseDuration = "120s"
-	leaderElectionRenewDeadline = "90s"
-	leaderElectionRetryPeriod   = "30s"
+	defaultLogLevel         = "info"
+	leaderElectionConfigMap = "hive-controllers-leader"
```

Comment:
Should this be 'leaderElectionLockName' (to follow the other changes in this PR)?

Reply:
That's done in the third commit. (You must be looking at the second.)

Reply:
Yep. Missed it as I hoped around the commits.